components/ContactCard/ContactCard.tsx

```diff
@@ -15,66 +15,64 @@ interface Props {
     isCompact?: boolean;
     renderAvatar: ({ className }: { className: string }) => ReactNode;
 }
-
 function ContactCard({ className, contact, isCompact = false, renderAvatar }: Props) {
     const device = useDevice();
-    const { name, description, company, email, phone, mobile, website } = contact;
+    const { name, description, company, email, phone, mobile } = contact;
     const { facebook, twitter } = getSocialHandles(contact);
     const subtitle = description && company ? `${description}, ${company}` : description;
-
     return (
         <div
             id={`contact-${contact.uuid}`}
             className={classNames(styles.container, className, {
                 [styles.compact]: isCompact || device.isMobile,
             })}
         >
-            {renderAvatar({ className: styles.avatar })}
             <div className={styles.content}>
```

Comment:
This wrapper is no longer necessary. The classname also doesn't exist anymore, so it does nothing. It's safe to remove it.